.circleci/config.yml

```diff
@@ -191,6 +191,12 @@ jobs:
             source ./.circleci/set_env_variables.sh
             cd java/app-encryption
             ./scripts/test.sh
+      - run:
```

Comment:
Think it should be ok to remove the test-app step from circleci in this PR, even if the actual directory is removed later